.github/workflows/startproject.yml

```diff
@@ -0,0 +1,83 @@
+name: Start Project
+
+on:
+    pull_request:
+    push:
+      branches: [main]
+
+jobs:
+  creates-valid-project:
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, '#skipci') }}
```

Comment:
issue: GitHub Actions [already has syntax for skipping CI](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs) which I think we should prefer for consistency

Reply:
I somehow missed that - good call.